<?xml version="1.0" ?>
<launch>
  <!--Define arguments for monitoring the master for the cameraman to follow-->
  <arg name="debug" default="true" />
  <arg name="world_frame" default="/map" />
  <arg name="base_frame" default="/base_link" />
  <arg name="right_left" default="true" />
  <arg name="num_stationary" default="5" />
  <arg name="loop_rate" default="1.0" />
  <arg name="tolerance" default="0.005" />

  <!--Wait for the selected link to stop moving for the required amount fo time-->
  <node name="cameraman" type="CameraMan.py" pkg="support_utilities" output="screen">
    <param name="debug" value="$(arg debug)"/>
    <param name="world_frame" value="$(arg world_frame)"/>
    <param name="base_frame" value="$(arg base_frame)"/>
    <param name="right_left" value="$(arg right_left)"/>
    <param name="num_stationary" value="$(arg num_stationary)"/>
    <param name="loop_rate" value="$(arg loop_rate)" />
    <param name="tolerance" value="$(arg tolerance)"/>
  </node>

  <node name="map_whiteout" type="map_whiteout.py" pkg="camerabot" output="screen">
  </node>

  <node name="vectorizer" type="vectorizer.py" pkg="camerabot" output="screen">
  </node>
</launch>
